apdfl18/Release_Notes.html

```diff
@@ -26,10 +26,10 @@ <h1>Release Notes, Adobe PDF Library v18</h1>
 <p><strong>APDFL v18.0.4PlusP2b</strong> (July 31, 2023)</p>
 <p>New Features:</p>
 <ul type="disc">
-<li>Updates open source PNG (libPNG) library to version 1.6.40.</li>
-<li>Updated the open source tiff library to version 4.5.1.</li>
 <li>SF#45777 - Improves support for PDF/A Parts 2 and 3 compliance with the addition of Level A options.</li>
 <li>SF#45826 – Adds a new method GetOutputPreviewImage() to the Page class of the .NET and Java interfaces.</li>
+<li>Updates open source PNG (libPNG) library to version 1.6.40.</li>
+<li>Updated the open source tiff library to version 4.5.1.</li>
 <li>With this release, the Java and .NET Interfaces version has been increased to 18.0.4.22.</li>
 <li>With this release, the Nuget package version has been increased to 18.22.0.</li>
```

Comment:
Actually version changes don't belong here either.

Reply:
I mean they would make more sense to be in a "Changes" section, otherwise under the heading of "new features" it doesn't quite make sense.

Reply:
"Changes" is fine - just didn't want to see this removed.

Reply:
👍